lispBM/c_libs/vesc_c_if.h

```diff
@@ -222,6 +222,13 @@ typedef enum {
 	VESC_PIN_HALL5,
 	VESC_PIN_HALL6,
 	VESC_PIN_PPM,
+#ifdef PIN_HW_1
```

Comment:
This enum should always have the pins defined even when not present on the hardware, otherwise things won't be aligned if more pins are added later. Otherwise the PR looks good.

Reply:
Thanks for the review, made it so the enum always defines the pins now.